### PR TITLE
use isWalkable concept rather than skipHopAnimation

### PR DESCRIPTION
**Pull request by Hamms**

### CODE REVIEW

File: src/js/game/LevelMVC/LevelBlock.js [hunk 1]
```diff
@@ -248,6 +248,10 @@ module.exports = class LevelBlock {
     return this.isEmpty || this.isEntity;
   }
 
+  static isWalkable(blockType) {
+    return new LevelBlock(blockType).isWalkable;
+  }
```
Comment (Hamms) [inline src/js/game/LevelMVC/LevelBlock.js]: @joshlory I'm not sure how I feel about this approach to exposing block-level concepts; I'd love to hear what you think about it

Comment (joshlory) [reply src/js/game/LevelMVC/LevelBlock.js]: Not a huge fan... can we use `block.isWalkable` below?

Ideally we'd stop referencing `blockType` at all in the View, but that will take a bit of cleanup.

**Comment by joshlory** [comment]
Cleanup looks great, but still on the fence about creating a new LevelBlock instance just to check some properties of a given `blockType`.  Seems like it would be much cleaner to always deal with `LevelBlock` instances in the View (but that will take a bit of cleanup).

File: src/js/game/LevelMVC/LevelBlock.js (hunk 1 above)
Comment (Hamms) [reply src/js/game/LevelMVC/LevelBlock.js]: Agreed that ideally we'd do that, but as it current stands there is nowhere in the call stack of this method that is actually creating a LevelBlock for us to call `isWalkable` on

**Pull request merged**